cmd/manager/main.go

```diff
@@ -193,26 +195,39 @@ func main() {
 }
 
 func initOperatorConfigMapVars(kubeClient client.Client) {
+	// Check if config map exists.
+	cm := &corev1.ConfigMap{}
+	err := kubeClient.Get(context.TODO(), types.NamespacedName{Namespace: awsv1alpha1.AccountCrNamespace, Name: awsv1alpha1.DefaultConfigMap}, cm)
+	if err != nil {
+		log.Error(err, "There was an error getting the default configmap.")
+		return
+	}
+
+	// Check if fedramp env
+	fr, ok := cm.Data["fedramp"]
+	if ok {
+		frBool, _ := strconv.ParseBool(fr)
+		if frBool {
+			aaoconfig.IsFedramp = frBool
```

Comment:
+1 with setting this once at start up instead of each reconcile. However, I still think we should hide the complexity in the config package like https://github.com/sam-nguyen7/aws-account-operator/commit/32652f2c7c2436025de2ae9aa06a29177a2d04a4#diff-435eecb1d2af40ee96747f88ab71eb2758da989c92f76dcb1f714fc1b300c633R186 . What do you think?

Reply:
i've cherry-picked this commit into this PR

thanks!